exchange/adapter_map.go

```diff
@@ -2,6 +2,7 @@ package exchange
 
 import (
 	"fmt"
+	"github.com/prebid/prebid-server/adapters/adpone"
```

Comment:
Can we put this include in the next paragraph where all the adapter imports are?
```
  3 import (
  4     "fmt"
  5  -  "github.com/prebid/prebid-server/adapters/adpone"
  6     "net/http"
  7     "strings"
  8
  9     "github.com/prebid/prebid-server/adapters"
 10     ttx "github.com/prebid/prebid-server/adapters/33across"
 11     "github.com/prebid/prebid-server/adapters/adform"
 12     "github.com/prebid/prebid-server/adapters/adkernel"
 13     "github.com/prebid/prebid-server/adapters/adkernelAdn"
 14  +  "github.com/prebid/prebid-server/adapters/adpone"
 15     "github.com/prebid/prebid-server/adapters/adtelligent"
 16     "github.com/prebid/prebid-server/adapters/advangelists"
exchange/adapter_map.go
```